src/orderbook.ts

```diff
@@ -1,6 +1,6 @@
 import {Order} from ".";
 
-export class Price {
+export class Fraction {
```

Comment:
If this fraction is being used to represent rational numbers, should this be using `BN` instead of plain `number`?